Assets/Plugins/Source/Core/EOSManager.cs

```diff
@@ -323,64 +323,50 @@ static void print(string toPrint)
             }
 
             //-------------------------------------------------------------------------
-            public void AddConnectLoginListener(IEOSOnConnectLogin connectLogin)
+            public void AddConnectLoginListener(IConnectInterfaceEventListener connectLogin)
```

Comment:
This is still changing the API from a `IEOSOnConnectLogin` type to a `IConnectInterfaceEventListner`, which changes the API signature.

Reply:
Makes sense - I reverted to the original signatures. The file still shows some changes, but they are whitespace changes and I think a re-ordering of the function definitions.